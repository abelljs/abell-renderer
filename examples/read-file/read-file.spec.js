const fs = require('fs');
const path = require('path');

const { buildAndGetSelector } = require('../../tests/utils/baseTestFramework');

describe('examples/read-file', () => {
  beforeAll(async () => {
    await buildAndGetSelector({
      exampleToRun: 'read-file',
      outPath: path.join(__dirname, 'out.html')
    });
  });

  it('should render csv data in out.html', () => {
    expect(
      fs.readFileSync(path.join(__dirname, 'out.html'), 'utf-8').trim()
    ).toEqual('hi, hello, nice\nhaha, woop, lol');
  });
});
