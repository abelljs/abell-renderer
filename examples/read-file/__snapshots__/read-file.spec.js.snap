// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`examples/read-file should render csv data in out.html 1`] = `
"hi, hello, nice
haha, woop, lol"
`;
